mlmodel/docs/Format/NeuralNetwork.rst

```diff
@@ -4170,7 +4220,7 @@ Output shape is same as the input.
 
 
 ReverseLayerParams
-________________________________________________________________________________
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
What does this do? 
This might go away when we switch to auto generating the rsts

Reply:
This is required for a level-3 subhead. For example, `NeuralNetwork` uses ==== for a level-1 head, Recurrent Layers uses ------ for a level-2 subhead, and ReverseLayerParams uses ^^^^^^ for a level -3 subhead. These are standard Sphinx/RST formatting codes.

Reply:
After auto-generating the content RSTs, they need to be edited anyway to fit into this organization. I have to provide a "map" and instructions for documenting this. Note that if I don't use these codes, the formatting is messed up and requires even more editing after generation.

Reply:
The only way to re-generate automatically from the protos is to organize the protos the same way, and to fix the comments in the protos to include formatting codes such as ====, -----, and ^^^^^.

Reply:
We need to make another pass through the protobuf files to edit the comments before relying on automatic RST generation. For now, we can publish this, and then schedule that work.